apps/site/assets/ts/components/RouteCardHeader.tsx

```diff
@@ -36,6 +38,8 @@ const RouteCardHeader = ({
           title="alert"
         >
```

Comment:
**suggestion:** try adjusting this link to look like a button via the Bootstrap classes -- you'll have to override some colors but I think by adding the "btn btn-sm" classes to this `<a>` and specifying a `border-color: currentcolor` (inherits the white text color), that can likely replace most of your added CSS.